lanebreach-api/app/models/sf311_case.rb

```diff
@@ -32,9 +32,17 @@
 #
 
 class Sf311Case < ApplicationRecord
+  has_one :sf311_case_metadatum
+
+  after_commit :add_metadata
```

Comment:
I think we may want to change this to an `after_create` callback so `Sf311CaseMetadatum` aren't created when existing `Sf311Case` records are changed. Thoughts?

Reply:
Additionally, we may want to consider adding a condition to this callback that specifically checks whether the `lat` or `long` attributes have changed.

Reply:
That makes sense about the `after_create` - I'll instead fix the callback to either a create new record or update an existing one.

I'm not expecting a `Sf311Case` object to get mutated after initial creation, so not sure if we need to add the `lat` and `long` _today_ checks but they make sense.